StadiumSeating/MainWindow.xaml.cs

```diff
@@ -9,102 +9,89 @@ namespace StadiumSeating
     public partial class MainWindow : Window
     {
         int classATotal = 0, classBTotal = 0, classCTotal = 0;
+        
         int classATickets = 0, classBTickets = 0, classCTickets = 0;
+        
         public MainWindow()
         {
             InitializeComponent();
-            tbTotalTicketsClassA.Content = "$" + classATotal.ToString("n2");
-            tbTotalTicketsClassB.Content = "$" + classBTotal.ToString("n2");
-            tbTotalTicketsClassC.Content = "$" + classCTotal.ToString("n2");
-            tbTotalRevenueGenerated.Content = "$" + 0.ToString("n2");
+            
+            tbTotalTicketsClassA.Content = classATotal.ToString("C");
+            
+            tbTotalTicketsClassB.Content = classBTotal.ToString("C");
+            
+            tbTotalTicketsClassC.Content = classCTotal.ToString("C");
+            
+            tbTotalRevenueGenerated.Content = 0.ToString("C");
```

Comment:
I changed the formatting to use the built-in "C" for formatting. This extends the usage of the app to other countries as Windows will automatically apply the appropriate currency signage according to their preferences.